packages/date/src/Date.js

```diff
@@ -181,9 +181,6 @@ const AvDate = ({
           onFocusChange={onFocusChange}
           numberOfMonths={1}
           isOutsideRange={isOutsideRange(min, max)}
-          customInputIcon={datepicker ? calendarIcon : undefined}
```

Comment:
Is there other code we can also remove now that the datepicker is being removed?

Reply:
I've gone back through and removed the unused code and imports, Thanks!